asmcomp/arm/emit.mlp

```diff
@@ -798,26 +798,48 @@ let emit_instr i =
 
 (* Emission of an instruction sequence *)
 
+(* consider the degenerate case where a single literal is followed by
+   a jump table longer than 4KB; we have to emit the constant pool
+   before the jump table or it will be too late *)
+let emit_instr_before_pool i =
```

Comment:
I think this comment could be improved.  Perhaps start with "It may be necessary to emit a constant pool before emitting the next instruction..." or something?